### ISSUE TITLE
Unsafe UPDATE query when updating customers_default_address_id in TABLE_CUSTOMERS

### ISSUE THREAD

**Issue by nikhilchaudhary70** [zencart/zencart#6890]
**Describe the bug**
Unsafe UPDATE query is being used in the class includes/classes/Customer.php:
```
    protected function setDefaultAddressBookId(int $id)
    {
        global $db;
        $sql = "UPDATE " . TABLE_CUSTOMERS . " SET customers_default_address_id=" . (int)$id;
        $db->Execute($sql);
        $this->data['customers_default_address_id'] = (int)$id;
    }
```

**Version**
Zen Cart version:  v1.5.8a
PHP version: 8.2
MySQL version: 8.1.31

**Comment by lat9** [comment]
Already handled in v2.0.0, see this commit: https://github.com/zencart/zencart/commit/92d3cb19324a24aa99161ad14f95e7d40a8e29da

**Comment by scottcwilson** [comment]
This is also in the known bugs list for 1.5.8/1.5.8a- see 

https://www.zen-cart.com/showthread.php?229042-Known-bugs-(and-fixes)-for-v1-5-8-series&p=1398572#post1398572
